### PR TITLE
Update bitops config templates to v2.0 schema

### PR DESCRIPTION
**Pull request by arm4b**
bitops.config schema updates to make it compatible with v2.0

### CODE REVIEW

File: generators/app/templates/terraform/bitops.config.yaml [hunk 1]
```diff
@@ -1,6 +1,4 @@
 terraform:
     cli: {}
     options:
-        command: apply
-        version: "0.13.2"
-        fetch-kubeconfig: false
\ No newline at end of file
+        stack-action: apply
```
Comment (arm4b) [inline generators/app/templates/terraform/bitops.config.yaml]: A note about moving `stack-action` under the `cli` parent per https://github.com/bitops-plugins/terraform/pull/17

Alternatively, we could just remove the option here, - that'll default to `plan`.
@PhillypHenning thoughts on that?

Comment (arm4b) [reply generators/app/templates/terraform/bitops.config.yaml]: Updated in https://github.com/bitovi/bitops-operations-repo-generator/pull/2/commits/e772b4ef79a8a87f36cdce4bfb341e5d9ac593c0 following your Terraform schema changes in `v2.2.0`

**Review by LeoDiazL — COMMENTED** [review]
Looking good. 
BitOps config files makes sense.

**Review by PhillypHenning — APPROVED** [review]

**Comment by arm4b** [comment]
Thanks!

Will need @mickmcgrath13  to push the new version to npm next.

**Pull request merged**